### ISSUE TITLE
Add CONTRIBUTING.md file 

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#75]
Add a `CONTRIBUTING.md` file in the root of the repository that contains this content:

```md
See the [Legesher contributing guide](https://github.com/legesher/legesher/blob/master/CONTRIBUTING.md)
```

Simple enough 👍 Please ask to be assigned to this issue so we don't have duplicates 😄

**Comment by locemarn** [comment]
can i get this one?

**Comment by madiedgar** [comment]
Absolutely @locemarn! I'll assign you to it!

**Comment by bhadreshpsavani** [comment]
Hello @locemarn and @madiedgar, Is it still open? 
Can I complete it?

**Comment by madiedgar** [comment]
Hey @bhadreshpsavani 👋 Thanks for checking up on this 👍 @locemarn has an open pr (#76), but it has some a minor change to be implemented. I'll give him one more day to work on the changes, and then I'll open up the issue again for you. 

In the meantime, if you know another language, we'd love to add it to our translations. 😄 I can walk you through it all!

**Comment by bhadreshpsavani** [comment]
Hello @mandiedgar,
I know Hindi and Gujarati as well, 
Can I help you with translation to this language?

**Comment by GauravMohla** [comment]
@madiedgar If needed, I can translate CONTRIBUTING file to German/French language (or both :))

**Comment by madiedgar** [comment]
oh wow!!! I hadn't thought of that @GauravMohla. Definitely would be something in the future, but we're still narrowing down the guidelines and wouldn't those changes to disrupt your translations.

However, would love your thoughts / translations for those languages in the `locale/` folder. See #42 and #43! You can review current translations or add new translations for other programming languages. All are helpful!

**Comment by GauravMohla** [comment]
> but we're still narrowing down the guidelines and wouldn't those changes to disrupt your translations.
Maybe I can look into translations once I the guidelines are formalized. 

I will try to add some translations to other programming language for sure :)

**Comment by chauxvive** [comment]
Hello, didn't look like anyone had done this still, so here it is. Happy #hacktoberfest!

**Comment by madiedgar** [comment]
Resolved in #107
